src/TgSharp.TL/TL/TLUser.cs

```diff
@@ -40,8 +40,8 @@ public override int Constructor
         public string LastName { get; set; }
         public string Username { get; set; }
         public string Phone { get; set; }
-        public TLAbsUserProfilePhoto Photo { get; set; }
-        public TLAbsUserStatus Status { get; set; }
+        [JsonIgnore] public TLAbsUserProfilePhoto Photo { get; set; }
+        [JsonIgnore] public TLAbsUserStatus Status { get; set; }
```

Comment:
I've realised this is an autogenerated file, so whenever we upgrade the TL Layer, we would lose these changes by accident; I'm wondering, can we create a new derived class that inherits from TLUser and marjs these two properties with [JsonIgnore]?

Reply:
What do you mean? I'm not sure about understanding your idea. Can you explain more in depth?

Reply:
```
public class UserForSession : TLUser {
    [JsonIgnore]
    public new TLAbsUserProfilePhoto Photo { get; set; }
    [JsonIgnore]
    public new TLAbsUserStatus Status { get; set; }
}
```

Reply:
Yes, that would probably work. Will try and commit this after your fix.

Reply:
Update: See latest commit message